test/2_create_test_app_namespace.sh

```diff
@@ -28,7 +28,12 @@ fi
 
 $cli delete --ignore-not-found rolebinding test-app-conjur-authenticator-role-binding-$CONJUR_NAMESPACE_NAME
 
-./k8s-config/test-app-conjur-authenticator-role-binding.sh.yml | $cli create -f -
+export TEST_DIR="k8s-config"
+if [[ "$PLATFORM" = "openshift" ]]; then
+    export TEST_DIR="openshift-config"
```

Comment:
what do you mean here? the hierarchy of the whole script or only from line 31+

Reply:
the hierarchy of folders. instead of having 2 folders name `something-config` we will have a `config` folder which will have 2 folders inside it - `k8s` & `openshift`

Reply:
on it!

Reply:
טיל @nessiLahav 🚀